### PR TITLE
Update node installation for required minimum version

### PR DESCRIPTION
**Pull request by mkangia**
## Product Description

No user facing change

## Technical Summary

Since our minimum requirement for node is 16.19.1, the setup command needs to be updated for it.

### Safety story

Just did a fresh setup and by using old command, I got dependency issues since it didn't meet our minimum requirements. Using the updated command all went well.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by SmittieC — APPROVED** [review]

File: DEV_SETUP.md [hunk 1]
```diff
@@ -632,7 +632,7 @@ On a clean Ubuntu 18.04 LTS install, the packaged nodejs version is v8. The
 easiest way to get onto the current nodejs v14 is
```
Comment (gherceg) [inline DEV_SETUP.md:632]:
```suggestion
easiest way to get onto the current nodejs v16 is
```
you could also change the line above this to reference ubuntu 22.04 LTS, and what appears to be v12 for the included nodejs version (good to confirm).

Comment (mkangia) [reply DEV_SETUP.md:632]: ah, I don't have the logs from my setup now.
I could update that to say
` clean Ubuntu 22.04 LTS install, the packaged nodejs version is expected to be v12`

Comment (gherceg) [reply DEV_SETUP.md:632]: Yeah that seems fine. It doesn't seem too important to specify which version of node is included in the standard ubuntu repositories anyway.

**Review by biyeun — APPROVED** [review]

File: DEV_SETUP.md (hunk 1 above)
Comment (mkangia) [reply DEV_SETUP.md]: Thanks @gherceg 
Made both fixes.

**Review by gherceg — APPROVED** [review]

**Pull request merged**